.travis.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     - stage: scala3
       name: scala3
       # separate job since only a few modules compile with Scala 3 yet
-      script: jabba install adopt@1.11-0 && jabba use adopt@1.11-0 && sbt -Dakka.build.scalaVersion=3.0 "akka-actor-tests/test:compile" akka-actor-testkit-typed/compile akka-actor-typed/compile akka-coordination/compile akka-pki/test:compile akka-protobuf/test:compile akka-protobuf-v3/test:compile akka-slf4j/test:compile akka-stream/compile akka-stream-tests-tck/test
+      script: jabba install adopt@1.11-0 && jabba use adopt@1.11-0 && sbt -Dakka.build.scalaVersion=3.0 "akka-actor-tests/test:compile" akka-actor-testkit-typed/compile akka-actor-typed/compile akka-discovery/compile akka-pki/test:compile akka-protobuf/test:compile akka-protobuf-v3/test:compile akka-slf4j/test:compile akka-stream/compile akka-stream-tests-tck/test akka-coordination/compile
```

Comment:
I tried to keep them alphabetically ordered to make merges simpler, but we can fix that on the next PR :)

Reply:
Oh, my thought the other way around. That it would be simpler for merge if they are always appended.

If we always append we can easily see which is the new module being added and just add it at the end again. As such we don't need to parse the string to find where is the difference in the middle of it.

Reply:
Hmm, that makes sense as well. I guess it just looked 'messy' to have them all mixed up, and tricky to see if we 'missed any'. But I don't have a strong preference.

Reply:
I was trying to append in an order such that there was as little transitive compilation as possible (compile root projects first). 😅

We need more opinions on the ordering of this compilation, please!

Reply:
PS: I'll redo my PR to add my project at the end. I think appending at the end simplifies things.